src/workerd/jsg/dom-exception.h

```diff
@@ -112,6 +113,15 @@ class DOMException: public Object {
     tracker.trackField("errorForStack", errorForStack);
   }
 
+  // The value is taken from worker-interface.capnp, which we can't depend
```

Comment:
I guess the reason we can't depend on it directly is because `DomException` is defined in `jsg` instead of under `api`?

Why is that? It kind of feels like it actually belongs under `api`, TBH. (Not that I'm suggesting changing it in this PR.)

Reply:
It really *should* be in `DOMException` and I just haven't had enough justification to move it. We certainly can tho.

Reply:
Added a comment. Will look at moving `DOMException` later.